skate-plugin/src/main/kotlin/com/slack/sgp/intellij/ui/FileChooserDescriptorUtil.kt

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2023 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.slack.sgp.intellij.ui
+
+import com.intellij.openapi.fileChooser.FileChooserDescriptor
+import com.intellij.openapi.vfs.VirtualFile
+
+object FileChooserDescriptorUtil {
+
+  fun createJarsChooserDescriptor() =
+    createMultipleFilesChooserDescriptor(allowJars = true) {
+      it.extension.equals("jar", ignoreCase = true)
+    }
+
+  fun createYamlChooserDescriptor() = createMultipleFilesChooserDescriptor {
+    it.extension.equals("yml", ignoreCase = true) || it.extension.equals("yaml", ignoreCase = true)
+  }
```

Comment:
Let's remove the things we're not using. Another reason to avoid copy-pasting unnecessarily :)

Reply:
Maybe I will just create a new kotlin file instead of the copied FileChooserDescriptorUtil.kt

Reply:
Did you mean to leave this after your update?